kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -1,93 +1,75 @@
 <template>
 
-  <icon-button @click="downloadContent" :text="downloadMediaText">
-    <svg src="download.svg"></svg>
-  </icon-button>
+  <div class="dropdown">
+    <button>Download Content &#9660</button>
```

Comment:
String should be in `$trs`, and then referenced here, so it will get translated.